doc/colorschemes.md

```diff
@@ -0,0 +1,77 @@
+# Colorschemes
```

Comment:
Could you switch back to the underline style headings, they're valid markdown and superior in plaintext form imo.